oaknational/Oak-Web-Application
# feat(lesson-counts): add correct lesson count for units with sensitiv…

…e and expired lessons

## Description

Music year: 1985

- List of changes

Unit Listing Page

1. Navigate to Year 7
2. Click Art and design
3. Confirm lesson count is displayed correct (not counting unavailable lessons) 

Lesson Listing Page

1. Navigate to Year 7
2. Click Art and design
3. Click Unit 2 Pop art
4. Confirm lesson count is displayed correctly (should be 3)


- [x]  New lessons heading count does not include is_sensitive lessons
- [x]  Lesson count in unit list component does not include is_sensitive lessons

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2542--oak-web-application.netlify.thenational.academy/pupils/programmes/art-secondary-year-7-l/units/pop-art-modern-popular-culture-4928/lessons
3. You should see correct lesson count

1. Go to https://deploy-preview-2542--oak-web-application.netlify.thenational.academy/pupils/programmes/rshe-pshe-secondary-year-9-l/units/
3. You should see correct lesson count


## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
